_how-it-works/laws-and-regulations/state-laws-and-regulations.md

```diff
@@ -29,6 +29,7 @@ breadcrumb:
     permalink: /how-it-works/
 title_display: State Laws and Regulations
 selector: list
+redirect_from: /how-it-works/state-legal-fiscal-info/
```

Comment:
We should add the [jekyll_redirect_from gem](https://github.com/jekyll/jekyll-redirect-from)

Make sure to `bundle` after or else it won't make it into the Gemfile.lock file that Federalist uses when it builds the site.